### ISSUE TITLE
Data Merge failures

### ISSUE THREAD

**Issue by firasm** [abelardopardo/ontask_b#99]
Hi @abelardopardo, first of all, thanks for all your work on this fantastic tool! Instructors on our campus are piloting it and giving us lots of positive feedback. I'm hoping you (or someone else) could help us track down a bug that's been super annoying and a real blocker to us promoting OnTask to more users on our campus.

**Bug:** System Error on merging data 

**Steps to reproduce:**

**- Super simple case**

1. Create a new workflow and import in `raw2.csv` in the attached testcases.zip file
2. In the Table Tab, under the Manage table data button, click "Upload or merge data"
3. Load in the same file (`raw2.csv`), skipping nothing.
4. Select Student number as the column to merge on
5. Select "Option 4" in the last question (Can also reproduce error with Option 3 selected).
6. Hit Finish twice, and you'll be greeted with a system error (which doesn't show up in the logs)
![screen shot 2018-10-12 at 5 27 56 pm](https://user-images.githubusercontent.com/2507459/46899292-4aa28e80-ce46-11e8-8f54-db769a7496e3.png)

**- Slightly more advanced case**

Repeat steps above, but use `raw2.csv` as the original file, and the `raw2_times100.csv`. The only difference in this file and raw2, is that Col2 is multiplied by 100.

If you could let us know if you can reproduce this issue or not, that might give us some clues as to what's going on.

Thanks a lot!

[testcases.zip](https://github.com/abelardopardo/ontask_b/files/2474929/testcases.zip)

**Comment by abelardopardo** [comment]
Hi. Thanks for the commens and for submitting the bug. A fix should be ready soon. Which version are you using?

**Comment by firasm** [comment]
I'm working on the same team as @andrew-gardener and @craigdsthompson over at UBC (https://github.com/ubc/ontask_b). We have moved to 3.0.1 in our testing environment but not yet in production.

The three bugs I reported (#97, #98, #99) are reproducible on v3.0.1 as well as 2.8.3.

FYI: Due to institution policies we aren't able to use emails for sending messages so have added some changes on top of each release in order to send messages through Canvas (#78). I've been told that we don't yet have resources to clean up our additions and push to the main codebase yet as we're in a pilot phase.

Thanks for agreeing to fix; I think the plan is that once this bug has been fixed, we'll apply it on our fork as well and push to production (even if it's not a full release) as the workarounds are quite time consuming.
